README.md

```diff
@@ -1,2 +1,56 @@
-# diagrams
-Utility tool to generate mermaid diagrams for Warehouse.ai
+# `@wrhs/diagrams`
+
+[![Version npm](https://img.shields.io/npm/v/@wrhs/diagrams.svg?style=flat-square)](https://www.npmjs.com/package/diagrams)
+[![License](https://img.shields.io/npm/l/@wrhs/diagrams.svg?style=flat-square)](https://github.com/warehouseai/diagrams/blob/master/LICENSE)
+[![npm Downloads](https://img.shields.io/npm/dm/@wrhs/diagrams.svg?style=flat-square)](https://npmcharts.com/compare/diagrams?minimal=true)
```

Comment:
`https://npmcharts.com/compare/diagrams?minimal=true` guessing that should have `@wrhs` in there as well.